spec/acceptance/certs_spec.rb

```diff
@@ -21,40 +23,37 @@
       apply_manifest(pp, catch_failures: true)
     end
 
-    describe command("/usr/bin/curl https://#{fact('hostname')}.example.com:443") do
-      its(:exit_status) { is_expected.to eq 60 }
+    specify do
```

Comment:
nice, never saw that before

Reply:
This is applying lessons from https://github.com/rubocop/rubocop-rspec/issues/1231. I had seen it, but not used it before.